gcp_variant_transforms/libs/bigquery_schema_to_vcf_header_test.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for `bigquery_schema_to_vcf_header` module."""
+
+import unittest
+
+from collections import OrderedDict
+
+from apache_beam.io.gcp.internal.clients import bigquery
+
+from vcf import parser
+
+from gcp_variant_transforms.beam_io import vcf_header_io
+from gcp_variant_transforms.libs import bigquery_schema_to_vcf_header
+from gcp_variant_transforms.libs import bigquery_util
+
+
+class VcfHeaderSourceTest(unittest.TestCase):
+  """Test cases for the `generate_header_fields_from_schema` function."""
+
+  def _get_table_schema(self):
```

Comment:
You might want to have this in a separate file than hard-coding it.  If the schema changes it'll become a headache with many changes or any transformations during migration.

Reply:
Thanks, Paul! It is a very good point, and I actually need this sample schema for testing in some other modules as well. Done!

Reply:
No problemo Allie, I was fairly sure you were aware of the abstraction :)